proto/eth/v2/BUILD.bazel

```diff
@@ -92,7 +96,9 @@ go_library(
 ssz_proto_files(
     name = "ssz_proto_files",
     srcs = [
+        "beacon_block.proto",
```

Comment:
want to make sure doing something like this makes sense, that beacon block can be moved here to make use of those variables for setting commitment length and blob length in proto